src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -343,14 +336,44 @@ let private createAnnotationTableAtRange (isDark:bool, tryUseLastOutput:bool, ra
     
             // Fail the function if there are not exactly 0 annotation tables in the active worksheet.
             // This check is done, to only have one annotationTable per workSheet.
-            let _ =
-                match annoTables.Length with
-                | x when x > 0 ->
-                    failwith "The active worksheet contains more than zero annotationTables. Please move to a new worksheet."
-                | 0 ->
-                    ()
-                | _ ->
-                    failwith "The active worksheet contains a negative number of annotation tables. Obviously this cannot happen. Please report this as a bug to the developers."
+            match annoTables.Length with
+            | 0 ->
+                ()
+            | x when x = 1 ->
+                //Create new worksheet and set it active
+                context.workbook.worksheets.add() |> ignore
+                let _ = context.workbook.load(propertyNames=U2.Case2 (ResizeArray[|"tables"|]))
+                let lastWorkSheet = context.workbook.worksheets.getLast()
+                lastWorkSheet.activate()
+                activeSheet <- lastWorkSheet
+                hasCreatedNewWorkSheet <- true
+            | x when x > 1 ->
+                failwith "The active worksheet contains more than one annotationTable. This should not happen. Please report this as a bug to the developers."
+            | _ ->
+                failwith "The active worksheet contains a negative number of annotation tables. Obviously this cannot happen. Please report this as a bug to the developers."
+        )
+
+        // Is user input signals to try and find+reuse the output from the previous annotationTable do this, otherwise just return empty array
+        let! prevTableOutput =
+            if (tryUseLastOutput) then getPrevTableOutput context                
+            else promise {return Array.empty}
+    
+        // If try to use last output check if we found some output in "prevTableOutput" by checking if the array is not empty.
+        let useExistingPrevOutput = (tryUseLastOutput) && Array.isEmpty >> not <| prevTableOutput
+
+        let! allTableNames = getAllTableNames context
+
+        let _ = activeSheet.load(propertyNames = U2.Case2 (ResizeArray[|"name"|]))
+
+        let newTableRange =
+            if hasCreatedNewWorkSheet then activeSheet.getCell(tableRange.rowIndex, tableRange.columnIndex)
```

Comment:
`activeSheet.getCell(tableRange.rowIndex, tableRange.columnIndex)`, why not get 0,0 (might be 1,1; excel is very inconsistent if they use 0 or 1 based indices).

Reply:
I wanted to keep the logic from before, when we create a child table, that is using the chosen index in the active sheet

Reply:
We need to see if the users like this or get confused, cant anticipate